admin/option_values.php

```diff
@@ -26,12 +26,34 @@
 switch ($_GET['action']) {
   case ('update_sort_order'):
     foreach ($_POST['options_values_new_sort_order'] as $id => $new_sort_order) {
-
-      $db->Execute("UPDATE " . TABLE_PRODUCTS_OPTIONS_VALUES . "
-                    SET products_options_values_sort_order = " . (int)$_POST['options_values_new_sort_order'][$id] . "
-                    WHERE products_options_values_id = " . (int)$id);
+            if (empty($_POST['language']) || $_POST['language'] == 'current_only') {
```

Comment:
Instead of having two nearly-identical queries here, it can probably be simplified with:
```php
if (empty($_POST['language']) || $_POST['language'] == 'current_only') {
    $languages_array = ['id' => $_SESSION['languages_id']];
} else {
    $languages_array = zen_get_languages();
}
foreach ($languages_array as $language) { 
...
```

Reply:
I've not looked at this since my original PR so no doubt there will be further improvements.

I meant a review of the results.